met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -8,7 +8,7 @@ import SurveyListing from 'components/survey/listing';
 import CreateSurvey from 'components/survey/create';
 import SurveyFormBuilder from 'components/survey/building';
 import SurveySubmit from 'components/survey/submit';
-import CommentReview from 'components/comments/CommentReview';
+import CommentReview from 'components/comments/admin/review/CommentReview';
```

Comment:
why the admin? we already mixed up public/internal stuff anyway

Reply:
Comments pages are comment review, comment listing (admin), comment text listing (admin), and comment text listing (public). When I was creating the file structure it made sense to separate them like that under the comments folder. What would you suggest instead?